Server/ReasnAPI/ReasnAPI/Services/ImageService.cs

```diff
@@ -108,9 +95,28 @@ public void UpdateImages(int objectId, List<ImageDto> imageDtos)
         }
 
         context.SaveChanges();
+    }
+
+    public void UpdateImage(int objectId, ImageDto imageDto)
+    {
+        if (imageDto is null)
+        {
+            throw new ArgumentException("No image provided");
+        }
+
+        var image = context.Images.Where(i => i.ObjectType == imageDto.ObjectType).FirstOrDefault(r => r.ObjectId == objectId);

```

Comment:
We can in where check for objectId to not get all images for this object type but just for this object type AND this object id

Reply:
i changed it for first or default with checking for object type AND object id